extensions/typescript-language-features/src/languageFeatures/quickFix.ts

```diff
@@ -336,7 +346,7 @@ class TypeScriptQuickFixProvider implements vscode.CodeActionProvider<VsCodeCode
 				expand = { kind: 'code-action', action };
 			}
 			else if (action.fixName === fixNames.fixMissingFunctionDeclaration && vscode.workspace.getConfiguration('typescript').get('experimental.aiCodeActions.missingFunctionDeclaration')) {
-				title += `Implement missing function declaration '${document.getText(diagnostic.range)}' using Copilot`;
+				title = `Implement missing function declaration '${document.getText(diagnostic.range)}' using Copilot`;
```

Comment:
looks like a few of these messages could be marked to be localized

Reply:
Good point. I'm working with the JS/TS team to share resources with an eventual VS version of these code actions, which will make it more complex, because the title will likely be loaded from a json file or something.